### DISCUSSION TITLE
Error starting with liionpack: AttributeError: 'Experiment' object has no attribute 'operating_conditions'

### DISCUSSION THREAD

**Discussion by bilgermoritz** [pybamm-team/liionpack#276] [Q&A] [answered]
Hi there,
I am getting this weird error while trying the getting started pages with liionpack. It tells me, that there is no attribute operating conditions in the experiment object. that makes no sense to me, because in that file there is never experiments.conditions called only an .'operating_conditions_steps'. I tried it in a normal python file instead of a jupyter notebook but got the same result.

It always comes up when i try starting the lp.solve(....).

**Comment by TomTranter** [comment, marked as answer]
Try an older version of pybamm. I haven't been maintaining liionpack much for a little while and operating conditions were removed. It's on my to do list to get things up to date soon

**Reply by bilgermoritz** [reply]
Sorry for the confusion, but it was a problem with my environment. So don't consider this as a problem anymore! Thank you for your fast response, i really appreciate the work you are doing.
